cmd/image-builder-tests/main_test.go

```diff
@@ -106,6 +105,42 @@ func RunTestWithClient(t *testing.T, ib string)  {
 	require.NoError(t, err)
 	require.Contains(t, []string{"pending", "running"}, composeStatus.ImageStatus.Status)
 
+	// A compose with gcp as a target
+	composeRequest = server.ComposeRequest{
+		Distribution: distro,
+		ImageRequests: []server.ImageRequest{
+			server.ImageRequest{
+				Architecture: arch,
+				ImageType: architectures[0].ImageTypes[0],
+				UploadRequests: []server.UploadRequest{
+					{
+						Type: "gcp",
+						Options: server.GCPUploadRequestOptions{
+							Bucket: "some-bucket",
+							ShareWithAccounts: &[]string{"somebody@example.com"},
```

Comment:
The account format must be one of:
- `user:{emailid}`
- `serviceAccount:{emailid}`
- `group:{emailid}`
- `domain:{domain}`

Plain email won't work.

Reply:
Ah ok, I'll change that. It doesn't matter hugely since it never actually completes the compose, it just makes sure the request sent is valid.